activerecord/test/cases/tasks/mysql_rake_test.rb

```diff
@@ -48,11 +56,17 @@ def test_establishes_connection_to_database
       ActiveRecord::Tasks::DatabaseTasks.create @configuration
     end
 
+    def test_when_database_created_successfully_outputs_info_to_stdout
+      $stdout.expects(:puts).with("Created database 'my-app-db'").once
```

Comment:
Let's make assertions about `$stdout.string` instead of mocking out the method calls we expect.